onnxruntime/core/providers/cpu/reduction/reduction_ops.cc

```diff
@@ -566,13 +565,10 @@ Status ReduceSum<T>::Compute(OpKernelContext* ctx) const {
 
   if (no_transpose) {
     const T* input_data = ctx->Input<Tensor>(0)->template Data<T>();
-
-#ifdef _OPENMP
-#pragma omp parallel for
-#endif
-    for (int64_t i = 0; i < block_size; ++i) {
+    auto lambda = [input_data, blocks, block_size, output_data](ptrdiff_t i) {
       output_data[i] = ConstEigenVectorMap<T>(input_data + (i * blocks), blocks).sum();
-    }
+    };
```

Comment:
For consistency should we do them all?

Reply:
Probably. For the objective of threadpool entry consolidation (removing all openmp calls), wanted to limit the scope of the changes here.